### DISCUSSION TITLE
Download translation file in a different format with wlc

### DISCUSSION THREAD

**Discussion by carlos-feliu-housfy** [WeblateOrg/weblate#9020] [Q&A]
I'm trying to downloads all translation files in different format that i created the project and it seems that it's not possible with webclient.

wlc download --output . -c laravel (or any other format)
Error: Object not found on the server (maybe operation is not supported on the server)

Is it possible?

Regards

**Comment by nijel** [comment]
See https://docs.weblate.org/en/latest/api.html#get--api-translations-(string-project)-(string-component)-(string-language)-file- for list of supported file formats for conversion.
